src/loadsave.jl

```diff
@@ -159,3 +159,51 @@ function loadtodict!(d::Dict, g::Union{JLDFile,Group}, prefix::String="")
     end
     return d
 end
+
+"""
+    save_object(filename, x)
+
+Write one object `x` from the current scope to a JLD2 file `filename`. The
+object is stored with a default name of "single_stored_object". If the default
+name needs to be overridden use [`@save`](@ref) instead. If the file exists, it
+overwrites it.
+
+# Example
+
+To save the string `hello` to the JLD2 file example.jld2:
+
+    hello = "world"
+    save_object("example.jld2", hello)
+"""
+function save_object(filename, x)
+  jldopen(filename, "w") do file
+    file["single_stored_object"] = x
```

Comment:
This situation with default object name is unique. Are we expecting similar situation to arise where default object name would be required? If the answer is yes, where would you recommend defining a module wide constant, in `loadsave.jl` or `JLD2.jl`?

Reply:
I'm not expecting a specific simular situation (although the test set already uses the default name, so one could argue that this counts a second use), I just prefer to keep the amount of non-trivial literals in generic code to a minimum.

Reply:
I'm not familiar enough with the structure of JLD2 to suggest where a potential constant should go.
@JonasIsensee any opinions?

Reply:
I think it would be good to define a constant like `constant SINGLE_OBJECT_NAME = "single_stored_object"`
directly above the source for `save_object` and `load_object`.
It is used nowhere else. 
Given that a name clash is unlikely and given that all modern IDEs support full-text search I wouldn't worry too much about it.